results/v1.19/known-test-failures.md

```diff
@@ -0,0 +1,12 @@
+# Known Test Failures for v1.19
+
+This page aims to summarize the causes of known test failures when running
+this version of Kubernetes conformance tests against a Supervisor cluster
+running vSphere Pods.
+
+- Test: `Kubectl cluster-info should check if Kubernetes master services is included in cluster-info`
+
+	Failure Reason: [Test has been updated to use inclusive terminology](https://github.com/kubernetes/kubernetes/commit/ab129349acadb4539cc8c584e4f9a43dd8b45761)
```

Comment:
Not clear about this.

Shouldn't the test have run correctly on supervisor cluster since the expected result has also been updated.

Reply:
So the test is looking for the older string. 
We are outputting the newer string.